tests/lib/rules/no-inner-literal.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-inner-literal');
+const {RuleTester} = require('eslint');
+
+let ruleTester = new RuleTester();
+ruleTester.run('no-inner-literal', rule, {
+  valid: [{
+    code: 'expect(a).to.be.ok;'
+  }, {
+    code: 'expect(a && b).to.be.ok;'
+  }, {
+    code: 'expect(a || b).to.be.ok;'
+  }, {
+    code: 'expect(a).to.equal(5);'
+  }],
+
+  invalid: [{
+    code: 'expect(undefined).to.be.ok;',
+    errors: [{
+      message: '"undefined" used in expect()'
+    }]
+  }, {
+    code: 'expect(null).to.be.ok;',
+    errors: [{
+      message: '"null" used in expect()'
+    }]
+  }, {
+    code: 'expect(true).to.be.ok;',
+    errors: [{
+      message: '"true" used in expect()'
+    }]
+  }, {
+    code: 'expect(52).to.be.ok;',
+    errors: [{
+      message: '"52" used in expect()'
+    }]
+  }, {
+    code: 'expect("string").to.be.ok;',
+    errors: [{
+      message: '""string"" used in expect()'
```

Comment:
this seems a bit unfortunate. what do you think about using backticks instead like in markdown?
```suggestion
      message: '`"string"` used in expect()'
```

Reply:
Done